Add link to Partykits privacy policy

mellson opened the pull request:
This PR adds a link to Partykits privacy policy.

laurakalbag commented on src/pages/privacy.mdx:
```diff
@@ -126,6 +126,14 @@ We use this information to detect and address any issues that may have been over
+## Connecting actors
+
+We use Partykit for live connections between running actors.
+We don't send any personal data to Partykit.
```
```suggestion
We don’t send any personal data to Partykit.
```

Yes, I am the worst.

laurakalbag reviewed (changes requested):
@mellson would you mind updating the line at the top of the privacy policy that has the date and what has changed? Otherwise I added a frankly pathetic nit suggestion.

mellson commented:
> @mellson would you mind updating the line at the top of the privacy policy that has the date and what has changed? Otherwise I added a frankly pathetic nit suggestion.

Thanks for the review, I've updated the doc
